core-web/apps/dotcms-ui/src/app/view/components/_common/dot-textarea-content/dot-textarea-content.component.ts

```diff
@@ -45,9 +45,18 @@ export class DotTextareaContentComponent implements OnInit, ControlValueAccessor
     @Input()
     width: string;
 
+    @Input()
+    hideBorder: boolean;
```

Comment:
```suggestion
    customStyles: Record<string, unkown>
```

And that way you let this ready to be extendable instead of only remove borders.